esphome/components/ethernet/ethernet_component.cpp

```diff
@@ -584,6 +588,50 @@ void EthernetComponent::ksz8081_set_clock_reference_(esp_eth_mac_t *mac) {
 
 #undef KSZ80XX_PC2R_REG_ADDR
 }
+
+void EthernetComponent::rtl8201_set_rmii_mode_(esp_eth_mac_t *mac) {
+#define RTL8201_RMSR_REG_ADDR (0x10)
```

Comment:
I just saw the same above and I will change that in its own pr to be a `constexpr uint8_t`
```suggestion
  constexpr uint8_t RTL8201_RMSR_REG_ADDR = 0x10;
```

Reply:
https://github.com/esphome/esphome/pull/6705